tomviz/modules/ModuleFactory.cxx

```diff
@@ -50,31 +50,38 @@ QList<QString> ModuleFactory::moduleTypes()
         << "Orthogonal Slice"
         << "Contour"
         << "Volume"
-        << "Threshold";
+        << "Threshold"
+        << "Molecule";
   qSort(reply);
   return reply;
 }
 
 bool ModuleFactory::moduleApplicable(const QString& moduleName,
                                      DataSource* dataSource,
+                                     MoleculeSource* moleculeSource,
```

Comment:
Why do we pass in both `dataSource` and `moleculeSource`? Should this method just be overloaded, not sure why we would need both?

Reply:
Looks like it would be clearer as an overload to me too.